### ISSUE TITLE
will a demo be released in the future?

### ISSUE THREAD

**Issue by Njasa2k** [OPPO-Mente-Lab/attention-mask-control#1]
is a demo release in the plans for the future?

**Comment by Wrch1994** [comment]
Sorry, due to company policy reasons, we currently have no way to open source our demo. However, in the latest commit, we have fixed some bugs in the training code and provided training scripts that have been tested and can be trained normally. Please follow the instructions in the readme to complete the training process yourself.
